test/replication/qsync_basic.result

```diff
@@ -374,7 +389,7 @@ test_run:switch('default')
  | ---
  | - true
  | ...
-box.cfg{replication_synchro_timeout = 0.001, replication_synchro_quorum = 3}
+box.cfg{replication_synchro_timeout = 0.1, replication_synchro_quorum = 3}
```

Comment:
Same as above, the smaller the timeout, the faster the test will finish. And there's no reason to wait more in this case

Reply:
fixed